go-controller/pkg/cni/ovs.go

```diff
@@ -92,6 +92,6 @@ func waitForBrIntFlows(ip string) error {
 		if err != nil {
 			return false, nil
 		}
-		return stdout!="", nil
+		return stdout != "", nil
```

Comment:
(This fix is in the second commit but should be squashed back into the first commit)

Reply:
yeah I just wanted to get CI results. I'll make sure that is fixed.